eclipse/src/saros/ui/wizards/pages/EnterProjectNamePage.java

```diff
@@ -110,35 +121,34 @@ public void performHelp() {
 
   @Override
   public void createControl(Composite parent) {
-    GridData gridData;
 
-    Composite composite = new Composite(parent, SWT.NONE);
-    setControl(composite);
+    Composite composite =
+        new Composite(parent, SWT.NONE) {
+          // dirty hack - if someone knows how to do it right with layout please change this
+          @Override
+          public Point computeSize(int wHint, int hHint, boolean changed) {
```

Comment:
Was this change/workaround included in this patch for convenience's sake or to counteract unwanted layout changes caused by the rest of this patch?

Reply:
It addresses the FIXME. TBH this class will be deleted somewhere in the near future. I do not see any reason to make small commits here and this change is described in the commit message. Of course it could be split.